package.json

```diff
@@ -13,6 +13,7 @@
   "sideEffects": false,
   "scripts": {
     "build-prep": "echo \"// This file is generated.\\nexport const version = '$npm_package_version'\" > src/generated/version.ts",
+    "version": "npm run build-prep",
```

Comment:
This change should cause `lerna` to update the version that's generated after bumping the package.